examples/hosts/iframe-host/src/outer.ts

```diff
@@ -83,25 +79,40 @@ async function getFluidObjectAndRender(container: Container, div: HTMLDivElement
     view.render(div, { display: "block" });
 }
 
-export async function loadDiv(divId: string) {
-    const div = document.getElementById(divId) as HTMLDivElement;
+async function loadOuterDataStoreAndLogDivs(
+    container: Container,
+    logDivId: string,
+    dataStoreDivId: string,
+): Promise<void> {
+    const logDiv = document.getElementById(logDivId) as HTMLDivElement;
 
-    const container = await getTinyliciousContainer(
```

Comment:
>container [](start = 10, length = 9)

this was fine, as this function is really just for testing to ensure we can had a div version of the same conntainer, and gives us another client to connect to the container in the iframe. it shouldn't reuse anything from the iframe case.